syntax/abstract.mjs

```diff
@@ -154,18 +159,36 @@ function attach_comments(acc, cur) {
     }
 }
 
-const LEADING_BLANK = /^[ \n\r]*/;
-const TRAILING_BLANK = /[ \n\r]*$/;
+// Remove the largest common indentation from a list of PatternElements.
```

Comment:
Extend this comment to make it expressive that all literal strings are indents, thanks to grammar.mjs separating those out?